### ISSUE TITLE
size_in_bytes in StoreStats can be larger than maximum Integer

### ISSUE THREAD

**Issue by fs-chris** [elastic/elasticsearch-java#433]
### Java API client version

8.4.3

### Java version

17

### Elasticsearch Version

8.4.3

### Problem description

 `size_in_bytes`, respectively all such values in `StoreStats` are defined as `int/Integer` but can be larger than the maximum integer.

Similar to #379 
```
Caused by: co.elastic.clients.json.JsonpMappingException: Error deserializing co.elastic.clients.elasticsearch._types.StoreStats: jakarta.json.stream.JsonParsingException: Jackson exception: Numeric value (16002499872) out of range of int (-2147483648 - 2147483647)
 at [Source: (org.apache.http.nio.entity.ContentInputStream); line: 1, column: 475] (JSON path: indices.store.size_in_bytes) (line no=1, column no=475, offset=-1)
	at App.war//co.elastic.clients.json.JsonpMappingException.from0(JsonpMappingException.java:134)
	at App.war//co.elastic.clients.json.JsonpMappingException.from(JsonpMappingException.java:121)
	at App.war//co.elastic.clients.json.ObjectDeserializer.deserialize(ObjectDeserializer.java:206)
	at App.war//co.elastic.clients.json.ObjectDeserializer.deserialize(ObjectDeserializer.java:136)
	at App.war//co.elastic.clients.json.JsonpDeserializer.deserialize(JsonpDeserializer.java:75)
	at App.war//co.elastic.clients.json.ObjectBuilderDeserializer.deserialize(ObjectBuilderDeserializer.java:79)
	at App.war//co.elastic.clients.json.DelegatingDeserializer$SameType.deserialize(DelegatingDeserializer.java:43)
	at App.war//co.elastic.clients.json.ObjectDeserializer$FieldObjectDeserializer.deserialize(ObjectDeserializer.java:71)
	at App.war//co.elastic.clients.json.ObjectDeserializer.deserialize(ObjectDeserializer.java:180)
	at App.war//co.elastic.clients.json.ObjectDeserializer.deserialize(ObjectDeserializer.java:136)
	at App.war//co.elastic.clients.json.JsonpDeserializer.deserialize(JsonpDeserializer.java:75)
	at App.war//co.elastic.clients.json.ObjectBuilderDeserializer.deserialize(ObjectBuilderDeserializer.java:79)
	at App.war//co.elastic.clients.json.DelegatingDeserializer$SameType.deserialize(DelegatingDeserializer.java:43)
	at App.war//co.elastic.clients.json.ObjectDeserializer$FieldObjectDeserializer.deserialize(ObjectDeserializer.java:71)
	at App.war//co.elastic.clients.json.ObjectDeserializer.deserialize(ObjectDeserializer.java:180)
	at App.war//co.elastic.clients.json.ObjectDeserializer.deserialize(ObjectDeserializer.java:136)
	at App.war//co.elastic.clients.json.JsonpDeserializer.deserialize(JsonpDeserializer.java:75)
	at App.war//co.elastic.clients.json.ObjectBuilderDeserializer.deserialize(ObjectBuilderDeserializer.java:79)
	at App.war//co.elastic.clients.json.DelegatingDeserializer$SameType.deserialize(DelegatingDeserializer.java:43)
	at App.war//co.elastic.clients.transport.rest_client.RestClientTransport.decodeResponse(RestClientTransport.java:328)
	at App.war//co.elastic.clients.transport.rest_client.RestClientTransport.getHighLevelResponse(RestClientTransport.java:294)
	at App.war//co.elastic.clients.transport.rest_client.RestClientTransport.performRequest(RestClientTransport.java:147)
	at App.war//co.elastic.clients.elasticsearch.cluster.ElasticsearchClusterClient.stats(ElasticsearchClusterClient.java:696)
```

**Comment by dixonqin** [comment]
I have solved this by copy the class to the same package path in your project, and change the fields to long.

**Comment by fs-chris** [comment]
This is fixed in version 8.7.1
